pypika/tests/test_terms.py

```diff
@@ -15,6 +15,18 @@ def test_when_alias_specified(self):
         self.assertEqual('bar', str(c1.alias))
 
 
+class FieldHashingTests(TestCase):
```

Comment:
Can you please add tests for when namespace isn't set also? Besides, can you please change your test to check whether the hash of field A is different from the hash of field B? Currently you are doing that in a very indirect way by checking the length of a dict, which have keys as fields. That definitely an over-engineered test, given you could just check whether the hash of them is equal/different.